src/prisma/_config.py

```diff
@@ -46,6 +41,27 @@ class DefaultConfig(BaseSettings):
         default=None,
     )
 
+    # Temporary workaround to support setting the binary platform until it can be properly implemented
+    binary_platform: Optional[str] = Field(env='PRISMA_BINARY_PLATFORM')
+
+    # Whether or not to use the global node installation (if available)
+    use_global_node: bool = Field(env='PRISMA_USE_GLOBAL_NODE', default=True)
+
+    # Whether or not to use the `nodejs-bin` package (if installed)
+    use_nodejs_bin: bool = Field(env='PRISMA_USE_NODEJS_BIN', default=True)
+
+    # Extra arguments to pass to nodeenv, arguments are passed after the path, e.g. python -m nodeenv <path> <extra args>
+    nodeenv_extra_args: List[str] = Field(
+        env='PRISMA_NODEENV_EXTRA_ARGS',
+        default_factory=list,
+    )
+
+    # Where to download nodeenv to, defaults to ~/.cache/prisma-nodeenv
+    nodeenv_cache_dir: Path = Field(
+        env='PRISMA_NODEENV_CACHE_DIR',
+        default_factory=lambda: Path.home() / '.cache' / 'prisma-nodeenv',
```

Comment:
If we are going to start owning our own cache (unless this is created by nodeenv), I think we should have the top level be our own package name similar to what prisma does with its `.prisma` directory. Same comment applies to the `'.cache' / 'prisma-binaries'` below

Reply:
Or re-use the same caching directory used by the upstream project to reduce the overhead of download for devs operating across both ecosystems. Not sure how common that is, but I fall into the category

Reply:
Changed the default cache directory to `~/.cache/prisma-python`, c1e76637845ae731a5ef8485fb98878dfeacb1b1.